extension/app/ts/components/subcomponents/CopyToClipboard.tsx

```diff
@@ -21,3 +21,21 @@ export function CopyToClipboard(props: Props): preact.VNode {
 		</div>
 	)
 }
+
+interface ToolTipParams {
+	children: preact.VNode
+	content: string
+}
+
+export function ToolTip(props: ToolTipParams): preact.VNode {
```

Comment:
The inferred type will likely be better than an explicit type here.
```suggestion
export function ToolTip(props: ToolTipParams) {
```